packages/reason-relay/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reason-relay",
-  "version": "0.9.2",
+  "version": "0.9.3",
```

Comment:
I think you can change this back to `0.9.2` for now. The next release will probably be `0.10.0` or so, not quite sure yet.

Reply:
haha sure, I forgot to revert this ^^

Reply:
😀